lib/generators/driver/templates/README.md

```diff
@@ -0,0 +1,3 @@
+## README
```

Comment:
I'm thinking it would be good to use the name of the driver as the title for the readme. We can use the `<%= class_name %>` snippet like in the initializer template.

https://github.com/degica/rails_drivers/blob/f54406e7f90ba3ac39f46a31f2dfa6127e359705/lib/generators/driver/templates/initializer.rb.erb#L1-L6

Reply:
makes sense to me. added